docs/CHANGELOG.asciidoc

```diff
@@ -44,7 +45,420 @@ This section summarizes the changes in each release.
+* Adds full custom metric UI to inventory alerts {kibana-pull}81929[#81929]
+* Adds basic interaction and shell for node details overlay {kibana-pull}82013[#82013]
+* Adds metrics to node details {kibana-pull}83357[#83357]
+* Don't show loading screen during auto-reload {kibana-pull}83376[#83376]
+* Adds logs to node details {kibana-pull}83433[#83433]
+* Adds Process tab to Enhanced Node Details {kibana-pull}83477[#83477]
+* Implements Resolved action group in Metrics alerts {kibana-pull}83687[#83687]
+* Adds metadata tab to node details flyout {kibana-pull}84454[#84454]
+* Fixes double loading of inventory page when there's a default view {kibana-pull}84843[#84843]
+* Synch time for metrics ui Kibana time {kibana-pull}85502[#85502]
+Monitoring::
+* Thread pool rejections alert {kibana-pull}79433[#79433]
+* Improves Collector `fetch` API {kibana-pull}79595[#79595]
+* Usage collection add saved objects client to collector fetch context {kibana-pull}80554[#80554]
+* Some progress on making alerts better in the UI {kibana-pull}81569[#81569]
+* CCR read exceptions alert {kibana-pull}85908[#85908]
+Operations::
+* Starting in 8.0, deb and rpm packages will restart on upgrade by default {kibana-pull}82049[#82049]
+* Adds a new CLI for generating encryption keys used by Kibana {kibana-pull}82838[#82838]
+* Adds deb and rpm packages for ARM64 {kibana-pull}84364[#84364]
+* Upgrades the major Node.js version used by and shipped with Kibana from v12 to v14 {kibana-pull}83425[#83425]
+* Adds experimental support for configuring CORS policy {kibana-pull}84316[#84316]
+Platform::
+* Apply back pressure in Task Manager whenever Elasticsearch responds with a 429 {kibana-pull}75666[#75666]
+* Adds basic observability into Task Manager's runtime operations {kibana-pull}77868[#77868]
+* Custom labels for ranges {kibana-pull}79628[#79628]
+* Adds cumulative sum expression function {kibana-pull}80129[#80129]
+* Adds a row click trigger to Lens embeddable for datatable expression renderer {kibana-pull}83167[#83167]
+* Adds search request batching using bfetch {kibana-pull}83418[#83418] and {kibana-pull}84043[#84043]
+* Normalize values by time unit {kibana-pull}83904[#83904]
+* Navigation search now includes deep links into various applications, allowing you to quickly navigate directly to the screens you need most {kibana-pull}83380[#83380]
+* Adds tags UI to search results {kibana-pull}85084[#85084]
+Querying & Filtering::
+* Use new Search API for rollup search {kibana-pull}83275[#83275]
+Security::
+* Adds audit logging events and event filtering{kibana-pull}74640[#74640]
+* Allow the default space to be accessed via `/s/default` {kibana-pull}77109[#77109]
+* Omit runtime fields from FLS suggestions {kibana-pull}78330[#78330]
+* Adds the ability to log in to Kibana anonymously without using any 3rd-party reverse proxy workarounds {kibana-pull}79985[#79985]
+* Filter out read access to config and telemetry obj {kibana-pull}82314[#82314]
```

Comment:
We can omit this one. It was a followup to an enhancement in this release, so it shouldn't be user-facing.

```suggestion
```